spring-cloud-dataflow-classic-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/ApiDocumentation.java

```diff
@@ -109,6 +109,7 @@ public void index() throws Exception {
 				linkWithRel("tasks/executions/name").description("Returns all task executions for a given Task name"),
 				linkWithRel("tasks/executions/execution").description("Provides details for a specific task execution"),
 				linkWithRel("tasks/platforms").description("Provides platform accounts for launching tasks"),
+				linkWithRel("tasks/platforms/scheduler").description("Provides platform accounts for launching tasks that have an associated scheduler"),
```

Comment:
I think it would be better to have a request parameter instead of path variable in this case. That way, we can do something like: `tasks/platforms?schedulesEnabled=true`

Reply:
I'm not really a big fan of using request parameters. If these are separate wouldn't it then be better to have `tasks/schedulers`.

Reply:
We can put it under tasks.  It just feels odd that a task would be returning platform information.   But equally it feels odd to have platforms returned that support scheduling.

Reply:
I think it comes down to when to use request parameter and when to use the path variable. I still believe using the request parameter would be a better fit here as we use this as a parameter to filter out the task platforms.

Reply:
👍

Reply:
Updated